extensions.json

```
@@ -243,6 +243,11 @@
       "download": "https://github.com/microsoft/vscode-eslint/releases/download/release%2F2.1.8/dbaeumer.vscode-eslint-2.1.8.vsix",
       "version": "2.1.8"
     },
+    {
+      "id": "deibit.devdocs",
+      "repository": "https://github.com/deibit/vscode-devdocs",
+      "version": "0.2.0"
```

> the repo doesn't have any git tags and its [`package.json`](https://github.com/deibit/vscode-devdocs/blob/master/package.json) doesn't include any packaging or publishing scripts

In that case, we don't need to pin a specific version, as we can always publish the latest version from the default branch.

```suggestion
      "repository": "https://github.com/deibit/vscode-devdocs"
```